python/lsst/daf/butler/core/schema.py

```diff
@@ -70,25 +45,38 @@ class Schema:
     Attributes
     ----------
     metadata : `sqlalchemy.MetaData`
-        The sqlalchemy schema description
-    dataUnits : `dict`
-        Columns that represent dataunit links.
+        The sqlalchemy schema description.
+    """
+    def __init__(self, config):
+        if isinstance(config, str):
+            config = SchemaConfig(config)
+        self.config = config
+        self.builder = SchemaBuilder()
```

Comment:
I only did that such that we could add more `DataUnit` tables later. But indeed, it might make more sense to separate it. Building is basically a one-off thing.